third_party/ienn/src/ie_compilation.cpp

```diff
@@ -257,11 +259,17 @@ int32_t Compilation::AddElementwise(const Operation& operation) {
     return result;
   for (size_t i = 0; i < 2; ++i) {
     const uint32_t input_index = operation.inputs[i];
-    // Setup constants
-    if (model_->values.find(input_index) != model_->values.end()) {
-      result = AddConstant(input_index);
-      if (result != error_t::NOT_ERROR) {
-        return result;
+    if (index_op_map_.find(input_index) == index_op_map_.end()) {
```

Comment:
Remove the `if ... else ...` so that elementwise operation can receive input node that is generated as last layer output.

Reply:
Every `node->output(0/1)` has been push into the `index_op_map_`, right? So I think elementwise operation can receive what it need as input nodes.